src/include/daos_srv/evtree.h

```diff
@@ -258,7 +260,7 @@ struct evt_list_entry {
 	struct evt_entry	 le_ent;
 };
 
-#define EVT_EMBEDDED_NR 32
+#define EVT_EMBEDDED_NR 16
```

Comment:
any reason to change this?

Reply:
if it does need bigger frame size, you can override the Sconscript like
denv.Append(CCFLAGS=['-Wframe-larger-than=8192'])

Reply:
This is for inline array size, not mandatory.

Reply:
so we have more than 4K in stack in one function, which is not very good, we have to pay more attention IMO.

Reply:
will this impact the performance?